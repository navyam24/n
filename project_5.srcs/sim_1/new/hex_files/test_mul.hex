45700093
0de00113
0000802b
0001002b
022081b3
0001802b
0000007b
