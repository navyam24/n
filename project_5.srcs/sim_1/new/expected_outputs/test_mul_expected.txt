out 0 00000000000000000000010001010111 00000457 1111
out 1 00000000000000000000000011011110 000000de 222
out 2 00000000000000111100001101110010 0003c372 246642
